index.html

```diff
@@ -122,6 +139,302 @@ <h5 class="color-purple-mountains-majesty" localization="why-myriad-distribution
+    
+
+                 <div class="platform">
+                			
+				         <b> Shop  </b> : 
+						 <p> &nbsp </p>
+	<a href="https://shop.spreadshirt.com/myriad-keepmyshirt/"><img src="images/trader 4.jpg" style="width:100%; height:100%" ></a>
+					
+					
+					<p> &nbsp </p>			   
+	<p> &nbsp </p>			   
+	
+	
+	
+	 <b> Blockfolio App</b> :
+<p> &nbsp </p>			   
+	<a href=""><img src="images/trader8.png" style="width:100%; height:100%" ></a>
+
+<p> &nbsp </p>			   
+	<p> &nbsp </p>			   
+	
+	
+	
+	 <b>Nutty.one</b>:
+<p> &nbsp </p>			   
+	<a href="https://myriad.nutty.one/"><img src="images/trader 11.jpg" style="width:100%; height:100%" ></a>
+
+		   
+	<p> &nbsp </p>			   
+	
+	<p> &nbsp </p>	
+	
+	 <b>Trezor.io</b>:
+<p> &nbsp </p>			   
+	<a href="https://trezor.io/coins/#XMY"><img src="images/trader 15.jpg" style="width:100%; height:100%" ></a>
+
+
+
+					</div>
+
+               					
+				
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+
+<!-- End code for new Merchants page >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> -->
+
+
+<!-- new code for Roadmap page >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> -->
+
+
+
+
+<div id="Road-map" class="section section-3 row">
+    <div class="col-xs-10 col-xs-push-1 col-sm-8 col-sm-push-2 col-md-6 col-md-push-3 text-center">
+      <h1 class="title" localization="why-myriad-section-header">R & D</h1>
+    </div>
+
+    <div class="col-xs-12 col-sm-10 col-sm-push-1">
+      <div class="panel">
+        <div class="panel-body">
+          <div class="row">
+		  
+		  
+            <div class="col-xs-12 col-sm-4">
+              <div class="info-group text-center">				
+				<script src='https://redditjs.com/subreddit.js' data-subreddit='myriadcoin' data-height='500' data-width='600' data-sort='top' data-theme='light' data-timeframe='month' data-subreddit-mode='large' ></script>
+				</div>		
+            </div>
+			
+		    <div class="col-xs-12 col-sm-4 text-center">
+              <div class="info-group">
+                <h5 class="color-coral-tree" localization="why-myriad-merge-mining-title"></h5>
+              </div>
+			  	
+			  </div>
+			  
+			  
+			   <div class="col-xs-12 col-sm-4 text-center">
+              <div class="info-group last">
+                
+                
+<a class="twitter-timeline" data-height="500" href="https://twitter.com/myriadcoin?ref_src=twsrc%5Etfw">Tweets by myriadcoin</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+	</div>
+
+               					
+				
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+
+<!-- End code for new Roadmap page >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> -->
```

Comment:
Not terribly fond of a "Roadmap" for fear of it being un-maintained. Leaning towards NAK here.

Reply:
the road map is just  `R&D from from embeded raddit  so it should update automatically, not sure if you seen the first draft. can see pv here: http://caveout.co.uk/myriad/xmy/Myriad.html